checkov/terraform/plan_utils.py

```diff
@@ -102,3 +102,7 @@ def get_entity_context(definitions, definitions_raw, definition_path, full_file_
                 entity_context['address'] = resource_defintion[TF_PLAN_RESOURCE_ADDRESS]
                 return entity_context
     return entity_context
+
+
+def get_resource_id_without_nested_modules(address: str) -> str:
+    return ".".join(address.split(".")[-4:])
```

Comment:
the random numbers are a bit wierd for anyone not familiar with the build, can you use a const?

Reply:
also here a doc string with an example, then it more clear, what actually is happening